spec/interactors/article/combine_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe Article::Combine do
+  let(:user) { User.find_by(login: OWNER) }
+  let!(:collection) { create(:collection, owner_user_id: user.id) }
+  let!(:work) { create(:work, collection: collection, owner_user_id: user.id) }
+  let!(:from_related_page) { create(:page, work: work, source_text: '[[Duplicate]]', source_translation: '[[Duplicate]]') }
+  let!(:from_source_article) do
+    create(:article, collection: collection.reload)
```

Comment:
What does `collection.reload` do here?

Reply:
at this point collection is an instance, after interactor the db changes have not been applied to this insttance of the object so we reload to requery from db the changes